genai-perf/genai_perf/export_data/console_exporter.py

```diff
@@ -46,6 +45,10 @@ def __init__(self, config: ExporterConfig):
         self._metrics = config.metrics
         self._config = config.config
 
+        # Set the maximum width of the 'Statistic' column.
+        # Any metric name+unit longer than this width will be wrapped.
+        self._max_width = 36
```

Comment:
Set the max width to 36, the length of the string for `Output Token Throughput (tokens/sec)` stat name. This was the longest name+unit string before this new TPS/user metric and so setting it to this length will not wrap for all the existing metrics. But any new metric that is newly introduced and is longer than this will be wrapped.

Reply:
Should this be a constant instead of magic number?

Reply:
I thought about it, and this won't likely be used or needed outside of this class so I just left it as an attribute. I could instead make it a class attribute, if you would like.